### DISCUSSION TITLE
how to automagically re-enable/restart a QR code mode after power off/power on the GoPro

### DISCUSSION THREAD

**Discussion by drivethebus** [gopro/labs#112] [General]
Hi - loving my gopro and this new QR code mode creator thingy.  I want the  Start via Motion Detection to be enabled even after I power off and then power up again (without having to re-scan the QR code).  Is this possible?

**Comment by catch-point** [comment]
Use the boot script.
See https://gopro.github.io/labs/control/extensions/
See https://gopro.github.io/labs/control/imutrigger/

**Comment by drivethebus** [comment]
okay, this is good, but when does it auto-upload when you have this configured?

**Comment by dnewman-gpsw** [comment]
No, the default auto-upload can't be using a boot command as it will take precedence.  If you want a scripted upload, you can do that, but it is not simple and may not be useful for your application.  See this example for a daily timelapse that uploads after each capture. https://gopro.github.io/labs/control/dailytl/
